ci/docker/x86_64-unknown-freebsd/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM rust-x86_64-unknown-freebsd
 
+# Building `aws-lc-rs` for Linux depends on `gcc-multilib`, `libclang` and `bindgen`.
```

Comment:
It feels confusing to me that we mention "Linux" in these FreeBSD/Illumos/NetBSD Dockerfiles. Maybe use the proper target OS name in the first line?

Should also suggest upstream that they clarify this page in their documentation that it applies to other Unix-likes too, ideally.

Reply:
@djc You're right. This info was extracted from their doc page and probably that very page needs to be updated.

Reply:
@djc Fixed on our side. I'll inform `aws-lc-rs` shortly.